aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -890,19 +891,20 @@ class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknownHtmlT
                     }
                 }
             }
-            imageGetter?.loadImage(it.getSource(), callbacks, this@AztecText.maxImagesWidth, this@AztecText.minImagesWidth)
+            imageGetter?.loadImage(it.getSource(), callbacks, maxDimension, maxDimension)
```

Comment:
replacing this call here with `imageGetter?.loadImage(it.getSource(), callbacks, maxImagesWidth, maxImagesWidth)` should work alright